tests/.suitespec.json

```diff
@@ -17,7 +17,21 @@
             "ddtrace/internal/*",
             "ddtrace/__init__.py",
             "ddtrace/py.typed",
-            "ddtrace/version.py"
+            "ddtrace/version.py",
```

Comment:
Not sure if this is the best approach but I think it's the safest. We can iterate on this in the future